tests/moe/test_b12x_w4a16_ep.py

```
@@ -0,0 +1,634 @@
+    if not torch.cuda.is_available():
+        return False
+    device = torch.device("cuda")
+    return is_sm120a_supported(device) or is_sm121a_supported(device)
+
+
+def _cuda_13_or_newer():
+    try:
+        from flashinfer.jit.cpp_ext import get_cuda_version
+
+        return get_cuda_version().major >= 13
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)